### ISSUE TITLE
Column resizer line not disappearing on column resize

### ISSUE THREAD

**Issue by rzhou186** [facebook/fixed-data-table#401]
My FixedDataTable's column resizer line is not hiding when I drag and release it, I've been trying to solve this for several hours and am not sure what is going wrong.

I've noticed that in online examples, whenever I release the resizer, an initially present class called `"fixedDataTableColumnResizerLineLayout_hiddenElem"` is re-added to the resizer via JavaScript, which hides the resizer.

That class isn't being re-added to my resizer. I've noticed that everything else about the resizer seems to match the online examples.

I've checked my own JavaScript, and don't think there is anything that might be interfering with FixedDataTable's own JavaScript for re-adding the class. For reference, I'm using an older React 0.14.6 (and the latest FixedDataTable 0.6.0). But looking at FixedDataTable's `package.json`, this seems like it shouldn't make a difference?

Thanks!

**Comment by AndreyStrib** [comment]
I can confirm. Having same issue.

**Comment by burakcan** [comment]
Any solutions?

**Comment by burakcan** [comment]
Found :) https://github.com/facebook/fixed-data-table/issues/167#issuecomment-118022854

**Comment by rzhou186** [comment]
Hey @burakcan - just read the issue you linked and it seems like the fix for one of the users was to "implement `isColumnResizing`"? Did anything in that issue fix it for you, and if so how did you do it? I was under the impression that `isColumnResizing` was implemented by FixedDataTable itself, not something that users of the library implement.

**Comment by rzhou186** [comment]
I figured it out. As it turns out, this isn't a bug - the answer is simply that if you are implementing a `Table` component with resizable columns, you must pass `isColumnResizing={false}` as `props` to that `Table`.

I realized this after reading the issue @burakcan linked above, as well as inspecting the source code for the FixedDataTable example that demonstrates resizable columns: https://github.com/facebook/fixed-data-table/blob/master/examples/ResizeExample.js#L61.

The documentation on `isColumnResizing` is really unclear: https://facebook.github.io/fixed-data-table/api-table.html. Both the name of the `prop` and the fact that it needs to be set to `false` don't make sense. Also, there is no in-depth documentation on it.
